vision/product_search/add_product_to_product_set.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package productsearch
+
+// [START vision_product_search_add_product_to_product_set]
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	vision "cloud.google.com/go/vision/apiv1"
+	visionpb "google.golang.org/genproto/googleapis/cloud/vision/v1"
+)
+
+func addProductToProductSet(w io.Writer, projectID string, location string, productID string, productSetID string) error {
```

Comment:
Please add a function comment.
```suggestion
// addProductToProductSet adds ...
func addProductToProductSet(w io.Writer, projectID string, location string, productID string, productSetID string) error {
```

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences